bnd.bnd

```diff
@@ -0,0 +1,32 @@
+Bundle-Activator: \
+    ${classes;IMPLEMENTS;org.osgi.framework.BundleActivator}
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
```

Comment:
This is IMHO added by bnd automatically and doesn't need to be declared explicitly

Reply:
Fixed in the latest changes.  Will do another pull request.